lib/common/options.c

```diff
@@ -557,61 +557,66 @@ pcmk__print_option_metadata(const char *name, const char *desc_short,
                             const char *desc_long,
                             pcmk__cluster_option_t *option_list, int len)
 {
+    /* approximate size of the text below, before adding all the options */
+    GString *s = g_string_sized_new(400);
```

Comment:
Lately I've been leaning to being generous with the initial allocation, since it's still relatively small, and can avoid a good bit of memory management overhead

Reply:
Fixed with a much more reasonable number.